sonic_platform_base/sonic_xcvr/api/amphenol/backplane.py

```diff
@@ -0,0 +1,89 @@
+"""
+    Custom API for Amphenol 800G Backplane Catridge
+"""
+import copy
+from ...fields import consts
+from ..public.cmis import CmisApi
+
+AMPH_BACKPLANE_INFO_DICT = {
+        "type": "N/A",
+        "type_abbrv_name": "N/A",
+        "hardware_rev": "N/A",
+        "serial": "N/A",
+        "cable_length": "N/A",
+        "manufacturer": "N/A", # Vendor Name
+        "model": "N/A", # Vendor Part Number
+        "vendor_date": "N/A",
+        "vendor_oui": "N/A",
+        "vendor_rev": "N/A",
+        **{f"active_apsel_hostlane{i}": "N/A" for i in range(1, 9)},
+        "application_advertisement": "N/A",
+        "host_electrical_interface": "N/A",
+        "media_interface_code": "N/A",
+        "host_lane_count": "N/A",
+        "media_lane_count": "N/A",
+        "host_lane_assignment_option": "N/A",
+        "media_lane_assignment_option": "N/A",
+        "cable_type": "N/A",
+        "media_interface_technology": "N/A",
+        "cmis_rev": "N/A",
+        "specification_compliance": "N/A",
+        }
+
+class AmphBackplaneImpl(CmisApi):
+    """
+    Custom API for Amphenol 800G Backplane Catridge
+    """
+    def __init__(self, xcvr_eeprom):
+        super(AmphBackplaneImpl).__init__(xcvr_eeprom)
+
+    def get_slot_id(self):
+        """
+        Get the slot id
+        """
+        slot = self.xcvr_eeprom.read(consts.CATRDIGE_SLOT_ID)
+        if slot is None:
+            return "NA"
+        return slot
+
+    def get_transceiver_info(self):
+        admin_info = self.xcvr_eeprom.read(consts.ADMIN_INFO_FIELD)
```

Comment:
@prgeor Can we instead call `super(CCmisApi, self).get_transceiver_info()` here to prevent code replication and then add the `AmphBackplaneImpl` specific fields here?

We can override `_get_xcvr_info_default_dict` function to define the default dictionary for `AmphBackplaneImpl`.
This could be similar to https://github.com/sonic-net/sonic-platform-common/blob/dd4b955387bcb04f9e733169bf7923b3a8b27dfd/sonic_platform_base/sonic_xcvr/api/public/c_cmis.py#L49-L55

Reply:
@mihirpat1  _get_xcvr_info_default_dict() will also read the connector field which is not applicable for this cable and we are using the same offset as slot ID instead. Hence this new override function for this specific cable